Material on managing access to the repository (#15)
empa-scientific-it/how-to-git: src/content/docs/01-repository.mdx

### Public vs Private repositories

When creating a repository, you can choose whether it will be public or private.
This choice affects who can see the content of the repository and affects how you can collaborate with others.

- **Public repository**: Anyone can see the content of the repository.
  This is useful for open-source projects or when you want to share your work with the world.
  The repository can be forked, which means that anyone can create a copy of the repository and work on it independently.
  People can also report issues, suggest changes, and contribute to the project.

> - **Private repository**: Only you (or the organisation members) and the people you invite can see the content of the repository.

Reviewer: What do you mean by organisation members?
```suggestion
- **Private repository**: Only you (the owner of the repository) and the people you invite can see the content of the repository.
```

Author: Right, I would then keep "the owner of the repo", without "or" - modified your suggestion.

  This is useful for personal projects or when you want to keep your work private.
  You can still invite people to collaborate on the repository, but they will need to have a GitHub account and be granted access explicitly.

> One can think of setting the visibility of the repository as an initial step to control access to the content and collaboration features.

Reviewer: ```suggestion
Setting the visibility of the repository is the initial step in controlling access to content and collaboration features.
```

> With more features, you can manage access to the repository in more detail, such as granting read or write access to specific people or teams.

Reviewer: ```suggestion
You can manage access to the repository in more detail using features, such as granting read or write access to specific people or teams.
```
